components/Sidebar/SidebarLatestStories/SidebarLatestStories.tsx

```diff
@@ -33,7 +33,7 @@ export const SidebarLatestStories = ({
     <Sidebar item elevated>
       <SidebarHeader>
         <Typography variant="h2">
-          <MenuBookRounded /> {label || 'Latest world news headlines'}
+          <MenuBookRounded /> {label || 'Latest from our partners'}
```

Comment:
@rpeterman-gp I figured this was a good a time as any to update this headline as editors requested.